### ISSUE TITLE
A tiny mistake in tiramisu-pytorch.ipynb

### ISSUE THREAD

**Issue by ashr59** [bfortuner/pytorch_tiramisu#2]
In **update_viz_summary_plot fuction**:

` 
def update_viz_summary_plot(self):

        trn_loss = self.loss_history['train'][-1]

        val_loss = self.loss_history['val'][-1]

        trn_err = self.error_history['train'][-1]    
    
        val_err = self.error_history['val'][-1]

        txt = ("""Epoch: %d

            Train - Loss: %.3f Err: %.3f

            Test - Loss: %.3f Err: %.3f""" % (self.epoch, 

            trn_loss, trn_err, tst_loss, tst_err))

        window = self.visdom_plots['summary']

        return viz.text(

            txt,

            win=window,

            env=self.name

        )
`
there are no **tst_loss** and **tst_err** variables.

 So I think they should be replaced by **val_loss** and **val_err** respectively, right?

**Comment by raghavab1992** [comment]
Yes i had this issue as well! Need to Replace it with val_loss and val_err.

**Comment by bfortuner** [comment]
Cleaned up the notebook and removed the visdom stuff for simplicity.
